src/test/java/com/microsoft/sqlserver/jdbc/resiliency/ResiliencyUtils.java

```diff
@@ -173,7 +174,35 @@ public void setTimer(int millis) {
         }
     }
 
+    public static Connection getConnection(String connectionString) throws SQLException {
+        Connection c = DriverManager.getConnection(connectionString);
+        minimizeIdleNetworkTracker(c);
+        return c;
+    }
+
+    public static void minimizeIdleNetworkTracker(Connection c) {

```

Comment:
Without this, we would need to either expose maxIdleMillis publicly or have tests sleep maxIdleMillis after killing connections in order to test the ICR feature. This allows us to test quickly.